ethereum/contracts/zksync/facets/Executor.sol

```diff
@@ -3,13 +3,13 @@
 pragma solidity 0.8.20;
 
 import {Base} from "./Base.sol";
-import {COMMIT_TIMESTAMP_NOT_OLDER, COMMIT_TIMESTAMP_APPROXIMATION_DELTA, EMPTY_STRING_KECCAK, L2_TO_L1_LOG_SERIALIZE_SIZE, MAX_INITIAL_STORAGE_CHANGES_COMMITMENT_BYTES, MAX_REPEATED_STORAGE_CHANGES_COMMITMENT_BYTES, MAX_L2_TO_L1_LOGS_COMMITMENT_BYTES, PACKED_L2_BLOCK_TIMESTAMP_MASK, PUBLIC_INPUT_SHIFT} from "../Config.sol";
```

Comment:
`MAX_REPEATED_STORAGE_CHANGES_COMMITMENT_BYTES` and `MAX_INITIAL_STORAGE_CHANGES_COMMITMENT_BYTES` can be removed from the `Config.sol` too btw